src/services/export.service.ts

```diff
@@ -10,17 +10,32 @@ import {
   defaultKeySheetBorderStyle,
   defaultKeySheetFontStyle,
 } from '../styles/keySheetStyles';
+import {
+  TestCaseExecutionResultDto,
+  TestCaseExcelExportDto,
+  PopulationDto,
+} from '@madie/madie-models';
 
 @Injectable()
 export class ExportService {
-  async generateXlsx(): Promise<Buffer> {
+  async generateXlsx(
+    testCaseExcelExportDtos: TestCaseExcelExportDto[],
+  ): Promise<Buffer> {
     const workbook = new ExcelJS.Workbook();
 
     //Generate Key worksheet
     const keyWorkSheet = workbook.addWorksheet('KEY');
     this.generateKeyWorksheet(keyWorkSheet);
 
     //Generate other worksheets as needed
+    const groupNumber = testCaseExcelExportDtos[0].groupNumber;
+    const populationWorksheet = workbook.addWorksheet(
+      `${groupNumber} - Population Criteria Section`,
+    );
+    this.generatePopulationWorksheet(
+      populationWorksheet,
+      testCaseExcelExportDtos[0],
+    );
```

Comment:
This is for single test case and single population for now and you are working on the multi right?

Reply:
@chubert-sb correct.